aiohue/v1/util.py

```diff
@@ -0,0 +1,24 @@
+"""Utils for aiohue."""
+from __future__ import annotations
+import logging
+
+
+def normalize_bridge_id(bridge_id: str):
```

Comment:
is this no longer necessary for v2 ? 

It's important for Home Assistant that the generated bridge IDs are the same after we upgrade this library or we should include a migration.

Reply:
that module was redundant, was also at toplevel. I removed the redundant one

Reply:
And yes, that normalize func is still in place